teuthology/task/install/rpm.py

```diff
@@ -179,28 +175,21 @@ def _update_package_list_and_install(ctx, remote, rpm, config):
                 cpack=cpack,
             )
             remote.run(
-                args=['if', 'test', '-e',
-                      run.Raw(pkg), run.Raw(';'), 'then',
-                      'sudo', pkg_mng_cmd, pkg_mng_opts, 'remove',
-                      pkg_mng_subcommand_opts, pkg, run.Raw(';'),
-                      'sudo', pkg_mng_cmd, pkg_mng_opts, pkg_mng_gpg_opt, 'install',
-                      pkg_mng_subcommand_opts, pkg_mng_install_opts,
-                      pkg, run.Raw(';'),
-                      'fi']
+                args=chain(
```

Comment:
@kshtsk could you please check `orchestra.run.quote()` to see if it accept a string like this?

Reply:
@tchaikov I don't think quote is used for bare multiline string, it is directed to the ssh session directly and works fine.

Reply:
@tchaikov 
```
>>> from teuthology.orchestra.remote import Remote as R
>>> R('127.0.0.1').run(args="""x=$(ls | grep 'teuthology')
... if test -e $x ; then
...     echo "File $x exists"
... fi""")
2018-10-25 14:44:00,891.891 INFO:teuthology.orchestra.run.127:Running: 'true'
2018-10-25 14:44:00,942.942 INFO:teuthology.orchestra.run.127:Running: 'x=$(ls | grep \'teuthology\')\nif test -e $x ; then\n\techo "File $x exists"\nfi'
2018-10-25 14:44:00,995.995 INFO:teuthology.orchestra.run.127.stdout:File teuthology.txt exists
RemoteProcess(client=<paramiko.client.SSHClient object at 0x1050f7ed0>, args='x=$(ls | grep \'teuthology\')\nif test -e $x ; then\n\techo "File $x exists"\nfi', check_status=True, hostname='127')
```

Reply:
makes sense! probably we can have a fix to improve the readability of `remote.run()` command. but i dare not do so at this moment, as it would imply large code churn. and we will need very careful about this sort of change.